src/components/kytos/misc/StatusMenu.vue

```diff
@@ -0,0 +1,764 @@
+          break;
+        case "PRE_SELECTED":
+          current_data = structuredClone(this.selectedSwitches)
+          break;
+        default:
+          current_data = structuredClone(this.switchData)
+          break;
+      }
+      //Filters Data based on current textFilter.
+      for (const element of this.switchTextFilter) {
+        if (element[1] != "") {
+          for (const s in current_data) {
+            var separatedFilter = element[1].toString()
+            var separatedProperty = current_data[s][element[0]].toString()
+            if (!(separatedProperty.includes(separatedFilter) || separatedFilter.includes(separatedProperty))) {
+              delete current_data[s]
+            }
+          }
+        }
+      }
+      return current_data
+    },
+    filtered_linkData: function() {
+      var current_data = {}
+      var updater = this.change
+      //Selects Data to filter from selectedLinks.
+      switch(this.currentLinkData) {
+        case "ORIGINAL":
+          current_data = structuredClone(this.linkData)
+          break;
+        case "PRE_SELECTED":
+          current_data = structuredClone(this.selectedLinks)
+          break;
+        default:
+          current_data = structuredClone(this.linkData)
+          break;
+      }
+      //Filters Data based on current textFilter.
+      for (const element of this.linkTextFilter) {
+        if (element[1] != "") {
+          for (const l in current_data) {
+            var separatedFilter = element[1].toString()
+            var separatedProperty = current_data[l][element[0]].toString()
+            if (!(separatedProperty.includes(separatedFilter) || separatedFilter.includes(separatedProperty))) {
+              delete current_data[l]
+            }
+          }
+        }
+      }
+      return current_data
+    },
+    filtered_interfaceData: function() {
+      var current_data = {}
+      var updater = this.change
+      //Selects Data to filter from selectedInterfaces.
+      switch(this.currentInterfaceData) {
+        case "ORIGINAL":
+          current_data = structuredClone(this.interfaceData)
+          break;
+        case "PRE_SELECTED":
+          current_data = structuredClone(this.selectedInterfaces)
+          break;
+        default:
+          current_data = structuredClone(this.interfaceData)
+          break;
+      }
+      //Filters Data based on current textFilter.
+      for (const element of this.interfaceTextFilter) {
+        if (element[1] != "") {
+          for (const i in current_data) {
+            var separatedFilter = element[1].toString()
+            var separatedProperty = current_data[i][element[0]].toString()
+            if (!(separatedProperty.includes(separatedFilter) || separatedFilter.includes(separatedProperty))) {
+              delete current_data[i]
+            }
+          }
+        }
+      }
+      return current_data
+    }
+  },
+  created() {
+    this.get_topology();
+    this.get_interfaces();
```

Comment:
As in the above comments, this won't be necessary

```suggestion
    this.get_topology();
```